.github/workflows/Func_Env_Test_CI.yml

```diff
@@ -85,8 +85,34 @@ jobs:
         pip install setuptools wheel twine
         python setup.py sdist bdist_wheel
         twine upload dist/*
-        echo '⌛ Wait 120 sec till package will be updated in PyPI'
-        sleep 120
+        echo '⌛ Wait till package will be updated in PyPI'
+        # Verfiy and wait till latest benchamrk runner version will be updated in Pypi (timeout 900 seconds)
+        timeout=900
+        sleep_time=30
+        expected_version=$(python3 setup.py --version)
+        pip --no-cache-dir install benchmark-runner --upgrade
+        build=$(pip freeze | grep benchmark-runner | sed 's/==/=/g')
+        actual_version="$(cut -d'=' -f2 <<<"$build")"
+        current_wait_time=0
+        while [[ $current_wait_time < $timeout ]];
+        do
+            if [ "$expected_version" == "$actual_version" ]; then
+            echo 'benchmark runner version: ' $expected_version 'was updated in Pypi'
+            break;
+            else
+            # sleep for x seconds
+            echo 'wait' "$((current_wait_time+$sleep_time))" 'seconds'
```

Comment:
$((current_wait_time+sleep_time))

variables inside $((...)) don't need the `$` prefix